### PR TITLE
updates readme

### PR DESCRIPTION
**Pull request by ELR143**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,47 @@
...
+    - "express": ^4.18.2
+    - "pg": ^8.7.3
+    - "pg-format": ^1.0.4
+
+5. Check the package.json file for the following **devDependencies**. Ensure that your versions meet the minimums below:
+    - "husky": ^8.0.2
+    - "jest": ^27.5.1
+    - "jest-extended": ^2.0.0
+    - "jest-sorted": ^1.0.14
+    - "supertest": ^6.3.3
```
Comment (Slyose) [inline README.md]: This isn't really a necessary step. When they clone your repo, this should all already be there as it is on main, so users won't have to manually check these dependencies

File: README.md [hunk 2]
```diff
@@ -1,8 +1,47 @@
...
 .env.test -> this should contain PGDATABASE=nc_news_test
 
+### Setup Instructions:
+
+1. In your terminal, navigate to your chosen directory, then copy and paste the following:
+`git clone https://github.com/ELR143/elr143-news-server.git`
+
+2. Run `npm install`
+
+3. Check your version of Node.js is v20.5.0 or higher --> `node -v`
```
Comment (Slyose) [inline README.md]: Also be worth mentioning the minimum version of pgsql! (you don't have to tell them how exactly to check these two, but you can :)  )

File: README.md [hunk 3]
```diff
@@ -1,8 +1,47 @@
...
+## Please add the following files to connect to the databases:
 
+.env.development -> this should contain PGDATABASE=nc_news
 .env.test -> this should contain PGDATABASE=nc_news_test
 
+### Setup Instructions:
+
+1. In your terminal, navigate to your chosen directory, then copy and paste the following:
+`git clone https://github.com/ELR143/elr143-news-server.git`
+
```
Comment (Slyose) [inline README.md:16]: If you're telling them to clone, it's also worth mentioning that they'll have to then navigate into the folder/repo with `cd elr143-news-server`

**Review by Slyose — COMMENTED** [review]
Left a few hints on how I'd perfect this before merging it, this is almost ready to be merged!

**Review by nc-mick-fay — CHANGES_REQUESTED** [review]
Just one more thing - add a link to your hosted server. then you're ready to merge!

**Review by nc-mick-fay — APPROVED** [review]
Ignore my last request, I didn't expand the top of the readme - merge away!

**Pull request merged**